analyzer/java/com/engflow/bazel/invocation/analyzer/suggestionproviders/NoCacheActionsSuggestionProvider.java

```diff
@@ -83,21 +107,28 @@ public SuggestionOutput getSuggestions(DataManager dataManager) {
                     "investigating actions that are not using remote caching"),
                 false));
       }
-      if (actionsWithoutRemoteCacheCheck.size() > maxActions) {
+      if (longEnoughActions.size() > maxActions) {
         caveats.add(
             SuggestionProviderUtil.createCaveat(
                 String.format(
                     "Only the %d longest actions that did not check the remote cache of the"
                         + " %d found were listed.",
                     maxActions, actionsWithoutRemoteCacheCheck.size()),
                 true));
+      } else if (longEnoughActions.size() < actionsWithoutRemoteCacheCheck.size()) {
+        caveats.add(
+            SuggestionProviderUtil.createCaveat(
+                String.format(
+                    "%d actions did not take long enough to be listed.",
+                    actionsWithoutRemoteCacheCheck.size() - longEnoughActions.size()),
```

Comment:
```suggestion
                    "%d actions did not check the remote cache, but %d of them were too fast to be listed.",
                    actionsWithoutRemoteCacheCheck.size(), actionsWithoutRemoteCacheCheck.size() - longEnoughActions.size()),
```

Reply:
This suggestion is attached to a suggestion, so I figured it was fine to drop the "no cache" clarification. Here's an example output generated:

```
Suggestion: "Investigate actions that are not using remote caching"
Some actions did not check the remote cache. Likely the targets the actions were executed for include the tag `no-cache` or `no-remote-cache`. Investigate whether these tags can be removed:
	- Action: "Copying directory external/npm/_/node_modules/date-fns [for tool]"
		Mnemonic: "CopyDirectory"
		Action duration: 52s
	- Action: "Copying directory external/npm/_/node_modules/date-fns"
		Mnemonic: "CopyDirectory"
		Action duration: 51s
	- Action: "Copying directory external/npm/_/node_modules/rxjs"
		Mnemonic: "CopyDirectory"
		Action duration: 46s
Rationale
The profile suggests remote caching was used, but some actions did not check the remote cache. In most cases, enabling remote caching for actions improves build performance. An example case where using remote caching might not be beneficial is when the outputs are very large and the the cost of downloading them is higher than locally building the target.
Caveats
- The profile does not include which target each action was processed for, although that data can help with investigating actions that are not using remote caching. It is added to the profile by using the Bazel flag `--experimental_profile_include_target_label`. Also see https://bazel.build/reference/command-line-reference#flag--experimental_profile_include_target_label
- 5201 actions did not take long enough to be listed. Try running this tool with --verbose to get more data.
```